### DISCUSSION TITLE
Dragonflight Local Network

### DISCUSSION THREAD

**Discussion by monteiroelias** [Burralis/Game-Launcher#42] [Q&A]
I would like to thank you in advance for the incredible work you did. I believe this is the place to definitively clear my doubt, so here goes:
 I have trinitycore compiled and I can make the connection without problems with the server and game on the same computer, but I would like to keep the server on a separate machine to gain more performance, is it possible without the need to acquire a domain and SSL certificate (both on local network)?

**Comment by Fabi** [comment]
> I would like to thank you in advance for the incredible work you did. I believe this is the place to definitively clear my doubt, so here goes: I have trinitycore compiled and I can make the connection without problems with the server and game on the same computer, but I would like to keep the server on a separate machine to gain more performance, is it possible without the need to acquire a domain and SSL certificate (both on local network)?

It works just fine on local network without a certificate. Otherwise you have to create a certificate.

**Reply by monteiroelias** [reply]
I don't understand what happened so I used the ip of my local network, 192.168.0.106. however whenever I tried to connect I was instantly disconnected.

**Reply by Fabi** [reply]
That is not an error related to the launcher.
